README.md

```diff
@@ -69,9 +69,14 @@ Function types are described by a **function signature**. The function signature
 To make the signature familiar to readers, we use common JavaScript idioms such as destructuring, defaults, and rest parameters:
 
 ```js
-({ count = 0: Number }) => Any
 (...args: String[]) => Any
 (myArray[]) => Any
+({ count = 0: Number }) => Any
+```
+
+If the type of the parameter is not an [union](#union-types), you can omit the type if it has a default value. Taking the previous signature as an example the type is derivable from the default value:
```

Comment:
Try:

If a parameter has a default value, most built-in types can be inferred:

```js
({ count = 0 }) => Any
```

If the type is too complex to infer from the default value, you should still specify the type:

```js
({ collection = []: Array | Object }) => Any
```